src/pages/Home/Home.page.jsx

```diff
@@ -1,39 +1,42 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import { $getVideos } from '../../api/videos';
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+import GridVideo from '../../components/GridVideo';
+import Container from '../../components/Common/Container';
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+const HomePageContainer = styled(Container)`
+  margin: auto;
+`;
+
+const HomePage = () => {
+  const [state, setState] = useState({
+    isLoading: true,
+    videos: [],
+  });
```

Comment:
Should not use object with multiple state values for `useState`, the component will re-render for all values even though only 1 value is changed
```
However, unlike this.setState in a class, updating a state variable always replaces it instead of merging it.
```